account.go

```diff
@@ -222,92 +239,62 @@ func (ea *ExternalAccount) UnmarshalJSON(b []byte) error {
 
 // LegalEntity is the structure for properties related to an account's legal state.
 type LegalEntity struct {
-	Type LegalEntityType `json:"type"`
+	Type LegalEntityType `json:"type" form:"-"` // Not updatable
```

Comment:
weirdly this *is* updatable and the library is likely wrong here (confirmed with curl)

Reply:
Ah yes, you're totally right. I think I must have thought that based on the old implementation. Fixed in 1171f1d.